README.md

```diff
@@ -1,11 +1,11 @@
-# Action for Installing MATLAB on GitHub-Hosted Runner
+# Action for Setting Up MATLAB on GitHub-Hosted Runner
 
-Before you run MATLAB&reg; code and Simulink&reg; models on a [GitHub&reg;-hosted](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners) runner, first use the [Set Up MATLAB](#set-up-matlab) action. The action installs the specified MATLAB release on a Linux&reg; virtual machine. If you do not specify a release, the action installs the latest release of MATLAB. 
+Before you run MATLAB&reg; code and Simulink&reg; models on a [GitHub&reg;-hosted](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners) runner, first use the [Setup MATLAB](#set-up-matlab) action. The action sets up the specified MATLAB release on a Linux&reg; virtual machine. If you do not specify a release, the action sets up the latest release of MATLAB.
 
-The **Set Up MATLAB** action is not supported on [self-hosted](https://docs.github.com/en/free-pro-team@latest/actions/hosting-your-own-runners/about-self-hosted-runners) runners. Currently, it is available only for public projects. It does not install transformation products, such as MATLAB Coder&trade; and MATLAB Compiler&trade;.
+The **Setup MATLAB** action is not supported on [self-hosted](https://docs.github.com/en/free-pro-team@latest/actions/hosting-your-own-runners/about-self-hosted-runners) runners. Currently, it is available only for public projects. It does not set up transformation products, such as MATLAB Coder&trade; and MATLAB Compiler&trade;.
```

Comment:
Not necessarily needed for this release, but I wonder if we should include a sentence after "The Setup MATLAB action is not supported..." sentence that says something like "To use MATLAB with a self-hosted agent, you can preintall MATLAB using the MATLAB installer."

Reply:
Interesting and I like that 🤔 😄 
@mw-hrastega would you happen to have a phrase in mind for something like this 🤔 
Else, we can defer it to a future release 😄

Reply:
I am not sure if we are on the same page. Why "can"? What is the other options? 

To use MATLAB with a self-hosted agent, you can preinstall MATLAB using the MATLAB installer.

Reply:
Okay we can PR this separately in time 😂